pyqtgraph/graphicsItems/ROI.py

```diff
@@ -213,7 +213,7 @@ def angle(self):
         """Return the angle of the ROI in degrees."""
         return self.getState()['angle']
         
-    def setPos(self, pos, update=True, finish=True):
+    def setPos(self, pos, y=None, update=True, finish=True):
```

Comment:
OK, make perfect sense now. May be it could be interesting to add your comment in the docstrings of the method as it will help people understand if (if the need to look deeply at the source code)